bower.json

```diff
@@ -1,8 +1,4 @@
 {
```

Comment:
Can this file go away completely since it is now empty?  Not sure.  I'm asking in the ember-cli slack group.